docs/deprecated_wiki/How-LORIS-URLs-work.md

```diff
@@ -1,32 +0,0 @@
-**[[HOME|Home]]** > **[[TECHNICAL|Technical]]** > **[[How LORIS URLs work]]**
```

Comment:
I don't think this document contains anything useful. Our routing is smart enough now that we don't need to specify front-slashes. This document is aimed at web-devs who should already have an understanding of GET requests and routing.